operator/pkg/images/images.go

```diff
@@ -77,7 +77,7 @@ const (
 var imageLookupMap map[Image]string = map[Image]string{
 
 	Cassandra_3_11_6: "datastax/cassandra-mgmtapi-3_11_6:v0.1.5",
-	Cassandra_3_11_7: "datastax/cassandra-mgmtapi-3_11_7:v0.1.13",
+	Cassandra_3_11_7: "datastax/cassandra-mgmtapi-3_11_7:v0.1.22",
```

Comment:
Thanks for the PR @zencircle.

The `v0.1.22` image is configured to run Cassandra as non-root whereas the `v0.1.13` image is configured to run as root. While running as non-root is definitely a good change, switching the default image in this way probably merits some discussion.

Reply:
@zencirle, John is right. You will likely run into problems with this bit of code:

https://github.com/zencircle/cass-operator/blob/patch-1/operator/pkg/images/images.go#L168-L179

ss it will assume that the default 3.11.7 image is run as root. The version you are upgrading too does not run as root as John said.

Reply:
@zencircle
You likely have to remove "3.11.7" from this line:
https://github.com/zencircle/cass-operator/blob/patch-1/operator/pkg/images/images.go#L172

and change the line in the test that expects it to be run as root (from false to true) here:
https://github.com/zencircle/cass-operator/blob/patch-1/operator/pkg/images/images_test.go#L70